test/util/enketo.js

```diff
@@ -24,6 +24,8 @@ const defaults = {
   // The number of times that the mock has been called during the test, that is,
   // the number of requests that would be sent to Enketo
   callCount: 0,
+  // An object with a property for each argument passed to the create() method
+  createData: undefined,
```

Comment:
I added `createData` instead of using `receivedUrl` in the new tests here, for two reasons:

- This allows us to test the `xmlFormId` argument of `enketo.create()` (the second argument), not just the first argument (`openRosaUrl`/`receivedUrl`).
- Consistency with `editData` elsewhere in this file, which we use to test `enketo.edit()`.